editor/scene_tree_dock.cpp

```diff
@@ -592,6 +593,9 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 
 			editor_data->get_undo_redo().commit_action();
 		} break;
+		case TOOL_NEW_SHADER: {
+			attach_shader_to_selected();
+		} break;
```

Comment:
This isn't really used anywhere (the option doesn't exist in menu), so you could just call `attach_shader_to_selected();` inside `open_shader_dialog()` and remove this option.

Reply:
Done